esphome/components/addressable_light/addressable_light_display.cpp

```diff
@@ -2,9 +2,9 @@
 #include "esphome/core/log.h"
 
 namespace esphome {
-namespace addressable_light_display {
+namespace addressable_light {
 
-static const char* TAG = "display.addressable_light_display";
+static const char* TAG = "display.addressable_light";
```

Comment:
Sorry, this is something that needs a little bit of tidy up in the whole repo, but it should be the other way around
```suggestion
static const char *TAG = "addressable_light.display";
```